Error during cargo install spl-token-cli

```
error[E0432]: unresolved imports `spl_token_v2_0::option`, `spl_token_v2_0::pack`
 --> /home/vladimir/.cargo/registry/src/github.com-1ecc6299db9ec823/solana-account-decoder-1.3.12/src/parse_token.rs:7:5
  |
7 |     option::COption,
  |     ^^^^^^ could not find `option` in `spl_token_v2_0`
8 |     pack::Pack,
  |     ^^^^ could not find `pack` in `spl_token_v2_0`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: failed to compile `spl-token-cli v2.0.1`, intermediate artifacts can be found at `/tmp/cargo-installWE6K8X`

Caused by:
  could not compile `solana-account-decoder`.

To learn more, run the command again with --verbose.
```

But `cargo build` in cli folder works for me.

```
➜  solana-program-library git:(master) cargo --version
cargo 1.46.0 (149022b1d 2020-07-17)
➜  solana-program-library git:(master) rustc --version
rustc 1.46.0 (04488afe3 2020-08-24)

```

--- a/token/cli/Cargo.toml
+++ b/token/cli/Cargo.toml
@@ -6,18 +6,18 @@ homepage = "https://spl.solana.com/token"
 license = "Apache-2.0"
 name = "spl-token-cli"
 repository = "https://github.com/solana-labs/solana-program-library"
-version = "2.0.1"
+version = "2.0.2"
 
 [dependencies]
 clap = "2.33.3"
 console = "0.12.0"
 serde_json = "1.0.57"
-solana-account-decoder = { version = "1.3.12" }
-solana-clap-utils = { version = "1.3.12"}
-solana-cli-config = { version = "1.3.12" }
-solana-client = { version = "1.3.11" }
-solana-logger = { version = "1.3.12" }
-solana-sdk = { version = "1.3.12" }
+solana-account-decoder = { version = "1.3.13" }
+solana-clap-utils = { version = "1.3.13"}
+solana-cli-config = { version = "1.3.13" }
+solana-client = { version = "1.3.13" }
+solana-logger = { version = "1.3.13" }
+solana-sdk = { version = "1.3.13" }
 spl-token = { version = "2.0", path="../program" }
 
 [[bin]]

--- a/token/cli/Cargo.toml
+++ b/token/cli/Cargo.toml
@@ -12,12 +12,12 @@ version = "2.0.2"
 clap = "2.33.3"
 console = "0.12.0"
 serde_json = "1.0.57"
-solana-account-decoder = { version = "1.3.13" }
-solana-clap-utils = { version = "1.3.13"}
-solana-cli-config = { version = "1.3.13" }
-solana-client = { version = "1.3.13" }
-solana-logger = { version = "1.3.13" }
-solana-sdk = { version = "1.3.13" }
+solana-account-decoder = { version = "=1.3.13" }
+solana-clap-utils = { version = "=1.3.13"}
+solana-cli-config = { version = "=1.3.13" }
+solana-client = { version = "=1.3.13" }
+solana-logger = { version = "=1.3.13" }
+solana-sdk = { version = "=1.3.13" }
 spl-token = { version = "2.0", path="../program" }
 
 [[bin]]

--- a/token/cli/src/main.rs
+++ b/token/cli/src/main.rs
@@ -120,7 +120,8 @@ fn command_create_token(
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
     check_fee_payer_balance(
         config,
-        minimum_balance_for_rent_exemption + fee_calculator.calculate_fee(&transaction.message()),
+        minimum_balance_for_rent_exemption
+            + fee_calculator.calculate_fee(&transaction.message(), None),
     )?;
     let mut signers = vec![
         config.fee_payer.as_ref(),
@@ -165,7 +166,8 @@ fn command_create_account(
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
     check_fee_payer_balance(
         config,
-        minimum_balance_for_rent_exemption + fee_calculator.calculate_fee(&transaction.message()),
+        minimum_balance_for_rent_exemption
+            + fee_calculator.calculate_fee(&transaction.message(), None),
     )?;
     let mut signers = vec![
         config.fee_payer.as_ref(),
@@ -198,7 +200,10 @@ fn command_assign(config: &Config, account: Pubkey, new_owner: Pubkey) -> Comman
     );
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref()];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);
@@ -244,7 +249,10 @@ fn command_transfer(
     );
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref()];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);
@@ -280,7 +288,10 @@ fn command_burn(config: &Config, source: Pubkey, ui_amount: f64) -> CommandResul
     );
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref()];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);
@@ -318,7 +329,10 @@ fn command_mint(
     );
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref()];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);
@@ -340,7 +354,10 @@ fn command_freeze(config: &Config, token: Pubkey, account: Pubkey) -> CommandRes
     );
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref()];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);
@@ -362,7 +379,10 @@ fn command_thaw(config: &Config, token: Pubkey, account: Pubkey) -> CommandResul
     );
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref()];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);
@@ -395,7 +415,10 @@ fn command_wrap(config: &Config, sol: f64) -> CommandResult {
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
     check_owner_balance(config, lamports)?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref(), &account];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);
@@ -427,7 +450,10 @@ fn command_unwrap(config: &Config, address: Pubkey) -> CommandResult {
     );
 
     let (recent_blockhash, fee_calculator) = config.rpc_client.get_recent_blockhash()?;
-    check_fee_payer_balance(config, fee_calculator.calculate_fee(&transaction.message()))?;
+    check_fee_payer_balance(
+        config,
+        fee_calculator.calculate_fee(&transaction.message(), None),
+    )?;
     let mut signers = vec![config.fee_payer.as_ref(), config.owner.as_ref()];
     unique_signers!(signers);
     transaction.sign(&signers, recent_blockhash);

